vantage6/node/docker/utils.py

```diff
@@ -8,6 +9,17 @@
 log = logging.getLogger(logger_name(__name__))
 
 
+def is_docker_running() -> bool:
+    """ Return True if docker engine is running"""
+    try:
+        client = docker.from_env()
+        client.ping()
+    except docker.errors.APIError as e:
+        log.error(e)
+        return False
+    return True
+
+
```

Comment:
Lets move this to the `vantage6.common` package and then replace the `check_if_docker_deamon_is_running` in both `vantage6.cli.node` and `vantage6.cli.server`